npm/react/examples/nextjs-webpack-5/cypress/plugins.js

```diff
@@ -1,3 +1,4 @@
+// @ts-check
 const injectNextDevServer = require('@cypress/react/plugins/next')
```

Comment:
Right now I see a handful of `setupDevServer`, along with a few individual `injectNextDevServer`, `injectWebpackDevServer`, `injectReactScriptsDevServer`, `injectWebpackWithBabelDevServer` names.

I also see the following types: `setupBabelDevServer`, `setupCRADevServer`, `setupCracoDevServer`, `setupNextDevServer`, `setupWebpackDevServer`.

Can we be more consistent with our names? We should either use `setupDevServer` everywhere, or we should reflect the exported type names (eg. `setupWebpackDevServer`) in our code and examples.

Reply:
I think consistency is a great idea. 
The reason why I have used `setupDevServer` is to simplify integration with the new syntax.
The reason why I use `inject` is to avoid mixing it with the setup ones.
The reason why I add the name of the plugin in the imported name is to make it clearer that it does more than setup the dev server. It also gathers information specific to a framework or a library.

Can you describe what advantages would come with having always the same function name?

Reply:
All the `setupDevServer` and `inject*` functions seem to be doing the exact same thing, which is to setup the dev server using `on`/`config`. Is there a reason they are being named differently? Because, if not, it till help to name them consistently to help reinforce the fact that they are all doing the same fundamental thing.

Reply:
Naming them consistently will help reinforce the fact that they are all doing the same basic thing, as far as the user is concerned. I'm concerned that people will be confused as to why some are named `setupDevServer`, the same name as the config property, while others are named like `injectReactScriptsDevServer`, which doesn't seem to correspond with anything in the config.

Reply:
I'd vote for replacing "injectWhatever" to become "setupWhatever" should be the naming convention. It's much more explicit and helps me understand what a function is specifically doing.

Reply:
How do you make the difference between a function that mutates the config and a functions that simply returns the running webserver?
One of the functions types actually changes the setup, the `inject` prefixed one.
The other starts the server and returns it, the `setup` one.

Naming is hard. 
It would be simpler for me to name all those functions `setupDevServer`, even if they do different things.
Is it what you guys mean?